lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -505,7 +505,7 @@ sub overview {
 
 sub latest {
     my ($self) = @_;
-    my %search_args;
+    my %search_args = (limit => 1);
```

Comment:
why is this necessary when "first" on the resultset should do the same? isn't the database executed just when "first" is called, not when the query object is created? As I understood complex_query returns the search object so also that should be ok. Isn't it?